common/views/components/IIIFSearchWithin/IIIFSearchWithin.tsx

```diff
@@ -180,63 +179,50 @@ const IIIFSearchWithin: FunctionComponent<Props> = ({
             const matchingCanvas = index && canvases[index];
             return (
               <ListItem key={i}>
-                <NextLink
-                  {...itemLink(
-                    {
-                      canvas: index || 0,
-                    },
-                    'search_within_result'
-                  )}
+                <ListLink
```

Comment:
Could we just make it of `type="button"` and we wouldn't need to `preventDefault()`?